addons/breathing/stringtable.xml

```diff
@@ -165,22 +165,11 @@
             <Portuguese>Remover Oxímetro de Pulso</Portuguese>
             <Finnish>Irrota pulssioksimetri</Finnish>
         </Key>
-        <Key ID="STR_KAT_Breathing_Pulseoximeter_Desc_Short">
+        <Key ID="STR_KAT_Breathing_Pulseoximeter">
             <English>Pulse Oximeter</English>
-            <German>Pulsoximeter</German>
-            <Polish>Pulsoksymetr</Polish>
-            <Spanish>Oxímetro de pulso</Spanish>
-            <Chinese>心率監測器</Chinese>
-            <Chinesesimp>脉搏血氧仪</Chinesesimp>
-            <Czech>Pulzní Oxymetr</Czech>
-            <Korean>맥박산소측정기</Korean>
-            <French>Oxymètre de pouls</French>
-            <Turkish>Nabız Oksimetresi</Turkish>
-            <Italian>Pulsossimetro</Italian>
-            <Russian>Пульсоксиметр</Russian>
-            <Japanese>パルスオキシメーター</Japanese>
-            <Portuguese>Oxímetro de Pulso</Portuguese>
-            <Finnish>Pulssioksimetri</Finnish>
```

Comment:
when i asked you to delete changed stringtables, i meant stuff that has more text than device name 😞